source/NVDAObjects/JAB/__init__.py

```diff
@@ -372,7 +384,8 @@ def _get_positionInfo(self):
 				return info
 
 		parent=self.parent
-		if isinstance(parent,JAB) and self.role in (controlTypes.Role.TREEVIEWITEM,controlTypes.Role.LISTITEM):
+		if isinstance(parent, JAB) and self.role in (
+			controlTypes.Role.TREEVIEWITEM, controlTypes.Role.LISTITEM, controlTypes.Role.TAB):

```

Comment:
Another linting change for readability 

```suggestion
		if (
			isinstance(parent, JAB)
			and self.role in (
				controlTypes.Role.TREEVIEWITEM,
				controlTypes.Role.LISTITEM,
				controlTypes.Role.TAB
			)
		):
```